### ISSUE TITLE
can't sign in

### ISSUE THREAD

**Issue by SuperDasher** [microsoft/vscode#117289]
Issue Type: <b>Bug</b>

I'm trying to sign in using my github account to sync settings and do other stuff with vscode, but even if I'm signed in with live share or github pull/push requests, the accounts icon on the bottom left still shows me notifications like I haven't signed in, and this issue causes sync to not work at all

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.94GB (6.87GB free)|
|Process Argv|--log debug --crash-reporter-id 1c30c0d7-ab10-4f4f-bdbd-528710ce738a|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (19)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|1.6.3
vscord|Leo|3.2.3
theme-monokai-pro-vscode|mon|1.1.18
python|ms-|2021.2.582707922
vscode-pylance|ms-|2021.2.3
jupyter|ms-|2021.2.576440691
vsliveshare|ms-|1.0.3784
java|red|0.75.0
synthwave-vscode|Rob|0.1.8
sonarlint-vscode|Son|1.20.1
tabnine-vscode|Tab|3.2.7
vscodeintellicode|Vis|1.2.11
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.18.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1
vscode-icons|vsc|11.1.0
material-theme|zhu|3.9.15

(9 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475

```

</details>

**Comment by RMacfarlane** [comment]
@SuperDasher Can you open the output panel and paste any content from the "GitHub Authentication" channel? Do you see any kind of error notification when you sign in?

**Comment by SuperDasher** [comment]
@RMacfarlane I've opened it, and the first thing I did was login to github to use liveshare and then do the same thing to activate settings sync. vscode immediately gives me an error when I open it and after every login, and it works fine, but settings sync gets turned off every time I close vscode because of this, aside from the fact that the icon keeps the notification as if I hadn't signed in. This is what I see: 
![Image](https://user-images.githubusercontent.com/72823509/109078674-04d87f00-76fe-11eb-9a64-1457c66840ad.png)

**Comment by RMacfarlane** [comment]
@SuperDasher Do you still see this in the latest release of VS Code? We made a fix to how credentials are persisted in 1.55 which may have fixed this

**Comment by SuperDasher** [comment]
Hey, sorry for not responding.
I just checked if it works with the updates, and it does, thanks for everything
